newsletter.md

```diff
@@ -0,0 +1,50 @@
+# ZK-newsletter for 2023-02-08
+Summary of the most important things, which happened in Zero Knowledge world, in the previous week.
+
+### New articles
+* [Dymension 35-C](https://medium.com/@dymensionXYZ/dymension-35-c-4dc8ac1c987c) - Dymension explains their first non-incentivized public testnet 35-C. (1.02.2023)
```

Comment:
This is already included in `Layer 2 projects`, so remove this. This link is announcment.